test/integration/job_type_test.py

```diff
@@ -28,11 +28,26 @@ class JobTypeTest(unittest.TestCase):
     staging_dir = os.environ['STAGING_DIR']
     working_dir = os.environ['WORKING_DIR']
     repository_dir = os.environ['REPOSITORY_DIR']
+    archaeocloud_dir = os.environ['ARCHAEOCLOUD_PATH']
+    BOOKSCAN_PATH = os.path.join(archaeocloud_dir, 'aronscans', 'bookscans')
```

Comment:
Warum einmal Konstanten und einmal Variablen?

Reply:
Und sollte man den Pfad, den man an `archaeocloud_dir` anhängt nicht besser aus `arachne_db_publish.py` importieren? Dann muss er nur an einer Stelle geändert werden, falls er sich mal ändert.

Reply:
Ich packe dir Pfade komplett in die Config, dann gibt es sie nur 1x und man muss sie nicht suchen.